test/Sentry.Tests/SentrySdkTests.cs

```diff
@@ -359,7 +359,7 @@ public void CaptureMessage_SdkInitialized_IncludesScope()
                 SentrySdk.AddBreadcrumb(expected);
                 _ = SentrySdk.CaptureMessage("message");
 
-                _ = worker.EnqueueEvent(Arg.Is<SentryEvent>(e => e.Breadcrumbs.Single().Message == expected));
+                //_ = worker.EnqueueEnvelope(Arg.Is<Envelope>(e => e.Breadcrumbs.Single().Message == expected));
```

Comment:
This deadcode should be gone or fixed. Follow up PR is fine, lets merge this!

Reply:
Completely missed it! Will fix.